abci/README.md

```diff
@@ -19,7 +19,7 @@ To get up and running quickly, see the [getting started guide](../docs/app-dev/g
 A detailed description of the ABCI methods and message types is contained in:
 
 - [The main spec](https://github.com/cometbft/cometbft/blob/v0.34.x/spec/abci/README.md)
-- [A protobuf file](./types/types.proto)
+- [A protobuf file](https://github.com/celestiaorg/celestia-core/blob/a12bfca93d748bd79da3c9966cdfb021b0cfc460/proto/abci/types.proto)
```

Comment:
I think it's better to relatively reference it instead of using a specific commit:

```suggestion
- [A protobuf file](../proto/abci/types.proto)
```

Reply:
Thanks, looks good.